README.md

```diff
@@ -315,16 +266,260 @@ For example:
+```
+
+### Create a foreign server with appropriate configuration:
+
+Once for a foreign datasource you need, as PostgreSQL superuser. Please specify database name using `dbname` option.
+
+```sql
+	CREATE SERVER influxdb_svr
+	FOREIGN DATA WRAPPER influxdb_fdw
+	OPTIONS (
+          dbname 'mydb',
+	  host 'http://localhost',
+	  port '8086'
+	);
+```
+
+### Grant usage on foreign server to normal user in PostgreSQL:
+
+Once for a normal user (non-superuser) in PostgreSQL, as PostgreSQL superuser. It is a good idea to use a superuser only where really necessary, so let's allow a normal user to use the foreign server (this is not required for the example to work, but it's secirity recomedation).
+
+```sql
+	GRANT USAGE ON FOREIGN SERVER influxdb_svr TO pguser;
+```
+Where `pguser` is a sample user for works with foreign server (and foreign tables).
+
+### User mapping
+
+Create an appropriate user mapping:
+```sql
+    	CREATE USER MAPPING
+	FOR pguser
+	SERVER influxdb_svr
+    	OPTIONS (
+	  user 'username',
+	  password 'password'
+	);
+```
+Where `pguser` is a sample user for works with foreign server (and foreign tables).
+
+### Create foreign table
+All `CREATE FOREIGN TABLE` SQL commands can be executed as a normal PostgreSQL user if there were correct `GRANT USAGE ON FOREIGN SERVER`. No need PostgreSQL supersuer for security reasons but also works with PostgreSQL supersuer.
+
+Create a foreign table referencing the InfluxDB table:
+You need to declare a column named `time` to access InfluxDB time column.
+```sql
+	CREATE FOREIGN TABLE t1(
+	  time timestamp with time zone,
+	  tag1 text,
+	  field1 integer
+	)
+	SERVER influxdb_svr
+	OPTIONS (
+	  table 'measurement1'
+	);
+```
+You can use `tags` option to specify tag keys of a foreign table.
+```sql
+	CREATE FOREIGN TABLE t2(
+	  tag1 text,
+	  field1 integer,
+	  tag2 text,
+	  field2 integer
+	)
+	SERVER influxdb_svr
+	OPTIONS (
+	  tags 'tag1, tag2'
+	);
+```
+You can import foreign schema
+```sql
+	IMPORT FOREIGN SCHEMA public
+	FROM SERVER influxdb_svr
+	 INTO public;
+```
+Access foreign table
+```sql
+	SELECT * FROM t1;
+```
+Limitations
+-----------
+
+- `UPDATE` is not supported.
+- `WITH CHECK OPTION` constraints is not supported.
+Following limitations originate from data model and query language of InfluxDB.
+- Result sets have different number of rows depending on specified target list.
+For example, `SELECT field1 FROM t1` and `SELECT field2 FROM t1` returns different number of rows if
+the number of points with field1 and field2 are different in InfluxDB database.
+- Timestamp precision may be lost because timestamp resolution of PostgreSQL is microseconds while that of InfluxDB is nanoseconds.
+- Conditions like `WHERE time + interval '1 day' < now()` do not work. Please use `WHERE time < now() - interval '1 day'`.
+
+When a query to foreign tables fails, you can find why it fails by seeing a query executed in InfluxDB with `EXPLAIN VERBOSE`.
+    
```

Comment:
Could you please add a Note section as the latest master?

----

## Notes
- InfluxDB FDW does not return an error even if it is overflow.
- EXP function of InfluxDB may return different precision number in different PC.

Reply:
Added.

Reply:
Thanks and sorry for my unclear comment.
This is not a Limitation, but added in a new Section called "Notes" in the original newly released Readme.
Do you have any idea of adding this information somewhere else?

Reply:
@weiting1lee , this notes look like limitations. First about ISO SQL:2016 overflow behaviour. Second about precision of some arithmetic operations in different processors. Some more neutral notes can be added in "Features" section.

Reply:
Yes, you're right.
Although, I felt they're a bit neutral, compare to put it in Feature section it's totally fit with Limitation section.
I'll close this conversation, thanks for answering.